### PR TITLE
Add password error msg to registration

### PR DESCRIPTION
**Pull request by auumgn**

### CODE REVIEW

**Comment by auumgn** [comment]
[Set a maximum length for passwords (3)](https://trello.com/c/oT9yFDcx/22-set-a-maximum-length-for-passwords-3)

File: src/app/register/components/form-password/form-password.component.html [hunk 1]
```diff
@@ -31,6 +31,12 @@
       >
         Password must match all pattern requirements
       </mat-error>
+      <mat-error
+        *ngIf="form.hasError('maxlength', 'password')"
+        i18n="@@register.passwordMaxLength"
+      >
+        Passwords must be 256 characters or less
```
Comment (leomendoza123) [inline src/app/register/components/form-password/form-password.component.html:38]: According to the designs this should read https://www.figma.com/file/dsrMiBCQsVngXCyjsoOLkw/Passwords-%26-validation?node-id=0%3A1

I have already added a tag for this `register.passwordIsToLongV2`

Comment (leomendoza123) [reply src/app/register/components/form-password/form-password.component.html:38]:
```suggestion
        Password must be between 8 and 256 characters
```

File: src/app/register/components/form-password/form-password.component.html [hunk 2]
```diff
@@ -31,6 +31,12 @@
       >
         Password must match all pattern requirements
       </mat-error>
+      <mat-error
+        *ngIf="form.hasError('maxlength', 'password')"
+        i18n="@@register.passwordMaxLength"
```
Comment (leomendoza123) [inline src/app/register/components/form-password/form-password.component.html:36]:
```suggestion
        i18n="@@register.passwordIsToLongV2"
```

File: src/locale/properties/register/register.en.properties [hunk 3]
```diff
@@ -22,6 +22,7 @@ register.passwordConfirmationRequired=Retype your password
 register.passwordNumber=1 number
 register.passwordSymbol=1 letter or symbol
 register.passwordLength=8 or more characters
+register.passwordMaxLength=Passwords must be 256 characters or less
```
Comment (leomendoza123) [inline src/locale/properties/register/register.en.properties:25]: ```suggestion```

File: src/locale/properties/register/register.lr.properties [hunk 4]
```diff
@@ -110,3 +110,4 @@ register.passwordIsToLongV2=LR
 register.passwordSymbolV2=LR
 register.passwordNumberV2=LR
 register.passwordConfirmationMatchV2=LR
+register.passwordMaxLength=LR
```
Comment (leomendoza123) [inline src/locale/properties/register/register.lr.properties:113]:
```suggestion
```

File: src/locale/properties/register/register.en.properties (hunk 3 above)
Comment (leomendoza123) [inline src/locale/properties/register/register.en.properties:25]: We will need to clean this strings as well

**Review by leomendoza123 — CHANGES_REQUESTED** [review]
A string update

File: src/locale/properties/register/register.rl.properties [hunk 5]
```diff
@@ -110,3 +110,4 @@ register.passwordIsToLongV2=RL
 register.passwordSymbolV2=RL
 register.passwordNumberV2=RL
 register.passwordConfirmationMatchV2=RL
+register.passwordMaxLength=RL
```
Comment (leomendoza123) [inline src/locale/properties/register/register.rl.properties:113]:
```suggestion
```

File: src/locale/properties/register/register.xx.properties [hunk 6]
```diff
@@ -110,3 +110,4 @@ register.passwordIsToLongV2=X
 register.passwordSymbolV2=X
 register.passwordNumberV2=X
 register.passwordConfirmationMatchV2=X
+register.passwordMaxLength=X
```
Comment (leomendoza123) [inline src/locale/properties/register/register.xx.properties:113]:
```suggestion
```

File: src/locale/properties/register/register.en.properties (hunk 3 above)
Comment (leomendoza123) [reply src/locale/properties/register/register.en.properties:25]:
```suggestion
```

**Pull request merged**